### ISSUE TITLE
How do you limit swipe direction?

### ISSUE THREAD

**Issue by broakenmedia** [ernestoyaquello/DragDropSwipeRecyclerview#9]
Seems to default to both directions but hoping for right to left only. Looks like something to do with the ListOrientation but cannot figure out the Java syntax.

**Comment by ernestoyaquello** [comment]
Unfortunately, **this is not currently possible**. However, I think it is a good idea, so I have implemented a way to do it that **will be available in the next release**.

In Kotlin, the syntax will be this one:
```kotlin
list.orientation?.removeSwipeDirectionFlag(ListOrientation.DirectionFlag.RIGHT)
list.orientation?.removeDragDirectionFlag(ListOrientation.DirectionFlag.UP)
```

In Java, even though I don't know for sure, I can imagine that the code will look somewhat like this:
```java
list.getOrientation().removeSwipeDirectionFlag(ListOrientation.DirectionFlag.RIGHT)
list.getOrientation().removeDragDirectionFlag(ListOrientation.DirectionFlag.UP)
```

I will try to do a new release this weekend.

Thanks for creating this issue!

**Comment by ernestoyaquello** [comment]
This feature is now included in the latest release: 0.0.3. You should be able to use it now :)

**Comment by broakenmedia** [comment]
Great stuff, thanks!
